projects/demo-playwright/tests/core/button/button.spec.ts

```diff
@@ -0,0 +1,21 @@
+import {TuiDocumentationPagePO, tuiGoto} from '@demo-playwright/utils';
+import {expect, test} from '@playwright/test';
+
+test.describe(`Button`, () => {
+    test(`tuiMode=OnDark + appearance=icon + hovered state`, async ({page}) => {
+        await tuiGoto(
+            page,
+            `/components/button/API?tuiMode=onDark&appearance=icon&icon=tuiIconEyeOff`,
+        );
+        const {apiPageExample} = new TuiDocumentationPagePO(page);
+
+        await apiPageExample.waitFor({state: `visible`});
```

Comment:
Use this type of assertions, please

https://playwright.dev/docs/test-assertions

Reply:
Thank you for the feedback, made the change accordingly